src/css/angular-tooltips.css

```diff
@@ -9,6 +9,8 @@
   -webkit-border-radius:3px;
   -moz-border-radius:3px;
   border-radius:3px;
+  left: -200%;
+  top: 0;
```

Comment:
@45kb hi, sorry. I missed here last time. 
I'm pretty sure 'top' is needed, because I got a bug in my project without it. Maybe it's Chrome's bug(it can't be reproduced on the other browsers). However, I think it's better to set top.